src/postprocessing/events.js

```diff
@@ -71,6 +71,35 @@ module.exports = {
         cleanTargetInterfaces(event, parsedInterfaces);
         delete event.spec;
         return event;
+      })
+      .sort((event1, event2) => {
+        if (event1.type < event2.type) {
```

Comment:
this could be simplified as `event1.type.localeCompare(event2.type) || event1.interface.localeCompare(event2.interface)`…

Reply:
Conceptually, I tend to avoid `localeCompare` because behavior depends on localization and the host's locale could well be different on a GitHub server and on our computers. Granted, that shouldn't change anything here given the characters used in `type` and `interface`. I'll switch to `localeCompare` but will make the locale explicit.